From 4b3070817f041c17acf70b126236d906dbad424b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?David=20D=C3=ADaz=20Gonz=C3=A1lez?= <dgonzalez@suse.de>
Date: Mon, 13 Jun 2022 22:31:45 +0100
Subject: [PATCH] Start loading a custom CSS file

For changing the Cockpit appearance using the CSS overrides workaround
described at https://github.com/cockpit-project/cockpit/pull/17437
---
 src/index.html | 1 +
 1 file changed, 1 insertion(+)

Index: cockpit-machines-292/src/index.html
===================================================================
--- cockpit-machines-292.orig/src/index.html
+++ cockpit-machines-292/src/index.html
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="stylesheet" href="index.css">
+    <link href="../../static/css-overrides.css" type="text/css" rel="stylesheet">
 
     <script type="text/javascript" src="index.js"></script>
     <script type="text/javascript" src="po.js"></script>
